thirdparty/README.md

```diff
@@ -528,7 +528,7 @@ File extracted from upstream release tarball:
   except `config_psa.h` and `psa_util.h`
 - All `.c` and `.h` from `library/` to `thirdparty/mbedtls/library/` except
   those starting with `psa_*`
-- The `LICENSE` file
+- The `LICENSE` file (edited to keep only the Apache 2.0 variant)
```

Comment:
I think mbedTLS used to be dual licensed Apache-2.0 OR GPL-2.0, then they removed the GPL-2.0 which was just forcing them to generate two sets of source tarballs for each license, and now they re-added the dual licensing...? Not sure what GPL-2.0 brings as an option next to Apache-2.0, but well.

We're obviously staying on the Apache-2.0 terms, so to prevent confusion, I kept the older `LICENSE` file which doesn't have the full text of the GPL-2.0 as an option.

Reply:
They added the GPLv2 because the FSF keeps saying that Apache 2.0 is not compatible with the GPLv2 (though they say it's compatible with the GPLv3).
There's some details here: https://www.apache.org/licenses/GPL-compatibility.html
And (briefly) in the release notes: https://github.com/Mbed-TLS/mbedtls/releases/tag/v2.28.6